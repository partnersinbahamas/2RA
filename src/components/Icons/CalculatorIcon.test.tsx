import React from 'react';
import { render } from '@testing-library/react';
import CalculatorIcon from './CalculatorIcon';

describe('CalculatorIcon', () => {
  it('renders small', () => {
    const { container } = render(<CalculatorIcon size="SMALL" />);
    expect(container).toMatchSnapshot();
  });
  it('renders medium', () => {
    const { container } = render(<CalculatorIcon size="MEDIUM" />);
    expect(container).toMatchSnapshot();
  });
  it('renders large', () => {
    const { container } = render(<CalculatorIcon size="LARGE" />);
    expect(container).toMatchSnapshot();
  });
  it('renders default', () => {
    const { container } = render(<CalculatorIcon />);
    expect(container).toMatchSnapshot();
  });
});
