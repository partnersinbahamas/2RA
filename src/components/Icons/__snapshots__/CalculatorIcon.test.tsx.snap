// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorIcon renders default 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="7 2 19 28"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M7 5C7 3.34315 8.34315 2 10 2H23C24.6569 2 26 3.34315 26 5V27C26 28.6569 24.6569 30 23 30H10C8.34315 30 7 28.6569 7 27V5ZM10 4C9.44772 4 9 4.44772 9 5V27C9 27.5523 9.44772 28 10 28H23C23.5523 28 24 27.5523 24 27V5C24 4.44772 23.5523 4 23 4H10Z"
    />
    <path
      d="M11 7C11 6.44772 11.4477 6 12 6H21C21.5523 6 22 6.44772 22 7V11C22 11.5523 21.5523 12 21 12H12C11.4477 12 11 11.5523 11 11V7Z"
    />
    <path
      d="M14 16.5C14 17.3284 13.3284 18 12.5 18C11.6716 18 11 17.3284 11 16.5C11 15.6716 11.6716 15 12.5 15C13.3284 15 14 15.6716 14 16.5Z"
    />
    <path
      d="M14 20.5C14 21.3284 13.3284 22 12.5 22C11.6716 22 11 21.3284 11 20.5C11 19.6716 11.6716 19 12.5 19C13.3284 19 14 19.6716 14 20.5Z"
    />
    <path
      d="M18 16.5C18 17.3284 17.3284 18 16.5 18C15.6716 18 15 17.3284 15 16.5C15 15.6716 15.6716 15 16.5 15C17.3284 15 18 15.6716 18 16.5Z"
    />
    <path
      d="M18 20.5C18 21.3284 17.3284 22 16.5 22C15.6716 22 15 21.3284 15 20.5C15 19.6716 15.6716 19 16.5 19C17.3284 19 18 19.6716 18 20.5Z"
    />
    <path
      d="M22 16.5C22 17.3284 21.3284 18 20.5 18C19.6716 18 19 17.3284 19 16.5C19 15.6716 19.6716 15 20.5 15C21.3284 15 22 15.6716 22 16.5Z"
    />
    <path
      d="M22 20.5C22 21.3284 21.3284 22 20.5 22C19.6716 22 19 21.3284 19 20.5C19 19.6716 19.6716 19 20.5 19C21.3284 19 22 19.6716 22 20.5Z"
    />
    <path
      d="M22 24.5C22 25.3284 21.3284 26 20.5 26C19.6716 26 19 25.3284 19 24.5C19 23.6716 19.6716 23 20.5 23C21.3284 23 22 23.6716 22 24.5Z"
    />
    <path
      d="M11 24C11 23.4477 11.4477 23 12 23H17C17.5523 23 18 23.4477 18 24V25C18 25.5523 17.5523 26 17 26H12C11.4477 26 11 25.5523 11 25V24Z"
    />
  </svg>
</div>
`;

exports[`CalculatorIcon renders large 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="7 2 19 28"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M7 5C7 3.34315 8.34315 2 10 2H23C24.6569 2 26 3.34315 26 5V27C26 28.6569 24.6569 30 23 30H10C8.34315 30 7 28.6569 7 27V5ZM10 4C9.44772 4 9 4.44772 9 5V27C9 27.5523 9.44772 28 10 28H23C23.5523 28 24 27.5523 24 27V5C24 4.44772 23.5523 4 23 4H10Z"
    />
    <path
      d="M11 7C11 6.44772 11.4477 6 12 6H21C21.5523 6 22 6.44772 22 7V11C22 11.5523 21.5523 12 21 12H12C11.4477 12 11 11.5523 11 11V7Z"
    />
    <path
      d="M14 16.5C14 17.3284 13.3284 18 12.5 18C11.6716 18 11 17.3284 11 16.5C11 15.6716 11.6716 15 12.5 15C13.3284 15 14 15.6716 14 16.5Z"
    />
    <path
      d="M14 20.5C14 21.3284 13.3284 22 12.5 22C11.6716 22 11 21.3284 11 20.5C11 19.6716 11.6716 19 12.5 19C13.3284 19 14 19.6716 14 20.5Z"
    />
    <path
      d="M18 16.5C18 17.3284 17.3284 18 16.5 18C15.6716 18 15 17.3284 15 16.5C15 15.6716 15.6716 15 16.5 15C17.3284 15 18 15.6716 18 16.5Z"
    />
    <path
      d="M18 20.5C18 21.3284 17.3284 22 16.5 22C15.6716 22 15 21.3284 15 20.5C15 19.6716 15.6716 19 16.5 19C17.3284 19 18 19.6716 18 20.5Z"
    />
    <path
      d="M22 16.5C22 17.3284 21.3284 18 20.5 18C19.6716 18 19 17.3284 19 16.5C19 15.6716 19.6716 15 20.5 15C21.3284 15 22 15.6716 22 16.5Z"
    />
    <path
      d="M22 20.5C22 21.3284 21.3284 22 20.5 22C19.6716 22 19 21.3284 19 20.5C19 19.6716 19.6716 19 20.5 19C21.3284 19 22 19.6716 22 20.5Z"
    />
    <path
      d="M22 24.5C22 25.3284 21.3284 26 20.5 26C19.6716 26 19 25.3284 19 24.5C19 23.6716 19.6716 23 20.5 23C21.3284 23 22 23.6716 22 24.5Z"
    />
    <path
      d="M11 24C11 23.4477 11.4477 23 12 23H17C17.5523 23 18 23.4477 18 24V25C18 25.5523 17.5523 26 17 26H12C11.4477 26 11 25.5523 11 25V24Z"
    />
  </svg>
</div>
`;

exports[`CalculatorIcon renders medium 1`] = `
<div>
  <svg
    class="sc-dmyCSP dRcSrL"
    fill="currentcolor"
    viewBox="3 1 17 22"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M3 4C3 2.34315 4.34315 1 6 1H17C18.6569 1 20 2.34315 20 4V20C20 21.6569 18.6569 23 17 23H6C4.34315 23 3 21.6569 3 20V4ZM6 3C5.44772 3 5 3.44772 5 4V20C5 20.5523 5.44772 21 6 21H17C17.5523 21 18 20.5523 18 20V4C18 3.44772 17.5523 3 17 3H6Z"
    />
    <path
      d="M6 5C6 4.44772 6.44772 4 7 4H16C16.5523 4 17 4.44772 17 5V7C17 7.55228 16.5523 8 16 8H7C6.44771 8 6 7.55228 6 7V5Z"
    />
    <path
      d="M9 10.5C9 11.3284 8.32843 12 7.5 12C6.67157 12 6 11.3284 6 10.5C6 9.67157 6.67157 9 7.5 9C8.32843 9 9 9.67157 9 10.5Z"
    />
    <path
      d="M9 14.5C9 15.3284 8.32843 16 7.5 16C6.67157 16 6 15.3284 6 14.5C6 13.6716 6.67157 13 7.5 13C8.32843 13 9 13.6716 9 14.5Z"
    />
    <path
      d="M13 10.5C13 11.3284 12.3284 12 11.5 12C10.6716 12 10 11.3284 10 10.5C10 9.67157 10.6716 9 11.5 9C12.3284 9 13 9.67157 13 10.5Z"
    />
    <path
      d="M13 14.5C13 15.3284 12.3284 16 11.5 16C10.6716 16 10 15.3284 10 14.5C10 13.6716 10.6716 13 11.5 13C12.3284 13 13 13.6716 13 14.5Z"
    />
    <path
      d="M17 10.5C17 11.3284 16.3284 12 15.5 12C14.6716 12 14 11.3284 14 10.5C14 9.67157 14.6716 9 15.5 9C16.3284 9 17 9.67157 17 10.5Z"
    />
    <path
      d="M17 14.5C17 15.3284 16.3284 16 15.5 16C14.6716 16 14 15.3284 14 14.5C14 13.6716 14.6716 13 15.5 13C16.3284 13 17 13.6716 17 14.5Z"
    />
    <path
      d="M17 18.5C17 19.3284 16.3284 20 15.5 20C14.6716 20 14 19.3284 14 18.5C14 17.6716 14.6716 17 15.5 17C16.3284 17 17 17.6716 17 18.5Z"
    />
    <path
      d="M6 18C6 17.4477 6.44772 17 7 17H12C12.5523 17 13 17.4477 13 18V19C13 19.5523 12.5523 20 12 20H7C6.44772 20 6 19.5523 6 19V18Z"
    />
  </svg>
</div>
`;

exports[`CalculatorIcon renders small 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="3 1 14 18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M3 4C3 2.34315 4.34315 1 6 1H14C15.6569 1 17 2.34315 17 4V16C17 17.6569 15.6569 19 14 19H6C4.34315 19 3 17.6569 3 16V4ZM6 3C5.44772 3 5 3.44772 5 4V16C5 16.5523 5.44772 17 6 17H14C14.5523 17 15 16.5523 15 16V4C15 3.44772 14.5523 3 14 3H6Z"
    />
    <rect
      height="3"
      rx="1"
      width="8"
      x="6"
      y="4"
    />
    <rect
      height="2"
      rx="1"
      width="2"
      x="6"
      y="8"
    />
    <rect
      height="2"
      rx="1"
      width="2"
      x="9"
      y="8"
    />
    <rect
      height="2"
      rx="1"
      width="2"
      x="12"
      y="8"
    />
    <rect
      height="2"
      rx="1"
      width="2"
      x="9"
      y="11"
    />
    <rect
      height="2"
      rx="1"
      width="2"
      x="12"
      y="11"
    />
    <rect
      height="2"
      rx="1"
      width="2"
      x="12"
      y="14"
    />
    <rect
      height="2"
      rx="1"
      width="2"
      x="6"
      y="11"
    />
    <rect
      height="2"
      rx="1"
      width="5"
      x="6"
      y="14"
    />
  </svg>
</div>
`;
